styles.css

```diff
@@ -0,0 +1,356 @@
+  display: flex;
+  justify-content: space-between;
+  max-width: 50%;
+  margin-left: 10px;
+  flex-direction: row;
+  list-style-type: none;
+}
+
+.recent-works {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: 100vh;
+  align-items: center;
+}
+
+.recent-works h1 {
+  margin-top: 30px;
+  font-family: "Crete Round", sans-serif;
+  color: #1c1a19;
+}
+
+hr {
+  width: 85%;
+}
+
+.head-work {
+  display: flex;
+  flex-direction: column;
+  min-height: 80vh;
+}
+
+.head-work img {
+  width: 80%;
+  align-self: center;
+}
+
+.head-work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.recent-works p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+}
+
+.techstack-container h3 {
+  font-family: 'Inter', sans-serif;
+  border: 1px solid #8993a4;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+}
+
+.head-work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: flex-start;
+  margin-left: 36px;
+  margin-top: 20px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+}
+
+.works {
+  display: grid;
+  background-color: #e5e5e5;
+  row-gap: 10px;
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  max-height: 80vh;
+  background-image: url(assets/Img\ Placeholder1.png);
+  background-repeat: no-repeat;
+  background-size: 90%;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-position: top;
+}
+
+.work h2 {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  width: 80%;
+  color: #fff;
+}
+
+.work p {
+  text-align: left;
+  margin-left: 36px;
+  margin-top: 24px;
+  max-width: 80%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.work .techstack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 36px;
+  max-width: 80%;
+  margin-top: 10px;
+}
+
+.work h3 {
+  color: #fff;
+}
+
+.work .project-button {
+  background-color: #ff6b00;
+  height: 48px;
+  color: #fff;
+  align-self: left;
+  margin-top: 22px;
+  margin-left: 1.5px;
+  padding: 12px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  font-family: 'Inter', sans-serif;
+  width: 99%;
+  border: none;
+}
```

Comment:
- Kindly make sure that you add an appropriate hover to your button, make the button (e.g., "Get my resume") a `link `a or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design) also do that for  The button (e.g., "See project"),  it will make your design interactive  to the Figma design as required in the project  :+1: